config.yaml

```diff
@@ -54,7 +54,9 @@ build_iso:
             - "open-power-host-os-virt-management"
         OpenPOWER Host OS RAS:
             - "open-power-host-os-ras"
-    iso_root_fs_packages_groups::
+    minimal_install_groups:
+        - "core"
```

Comment:
- I'd prefer to make it clear in var name this belongs to base distro (CentOS), hard to know these groups belongs to the base distro (Cent OS) and not OP Host OS without reading help below. Maybe you consider base_distro_minimal_install_groups is too long? Up to you.
- I'd prefer to refer to packages groups consistently. Where here core does not have a @ and below it does?